python/hpsmc/tools.py

```diff
@@ -713,9 +722,56 @@ def cmd_args(self):
         args.append(self.java_class)
         return args
 
+class EvioToLcioConversion(JavaTool):
```

Comment:
Why are we adding a component that appears to do the same thing as a component we already have? In what way is this unique with respect to EvioToLcio?

Reply:
The tool EvioToLcio is to convert evio to lcio, and run a steering-file job. Construction of command args in cmd_args() is not suitable for only evio-to-lcio conversion with EvioToLcio command (needed for MC with pulser data merging). So I added another tool with different construction of command args. I think the solution is a good choice not to interrupt job scripts we have built.

Reply:
In what way is it not suitable? I have done a lot of "only evio-to-lcio conversion" with EvioToLcio without any issue. I think the solution already available with EvioToLcio is sufficient.

Reply:
This is the steering file available in hps-java for doing exclusive evio-to-lcio conversion of data: https://github.com/JeffersonLab/hps-java/blob/master/steering-files/src/main/resources/org/hps/steering/production/Evio2LcioCnv.lcsim

Reply:
The good choice is to use the tools already available to achieve this.

Reply:
Okay. Good to know that we have another way for conversion with steering file. Previously, I just followed instructions in https://confluence.slac.stanford.edu/display/hpsg/Running+HPS+Java to run hps-java for conversion. 
I agree the way you suggested is better. Let me do modification and tests.